src/authplus.ts

```diff
@@ -20,4 +28,37 @@ export class AuthPlus extends GoogleAuth {
   Compute = Compute;
   // tslint:disable-next-line: variable-name
   OAuth2 = OAuth2Client;
+  // tslint:disable-next-line: variable-name
+  GoogleAuth = GoogleAuth;
```

Comment:
So this class *extends* `GoogleAuth`, and we're adding a reference to `GoogleAuth` itself?  My head is spinnin a bit.

Reply:
I'm adding a reference to `GoogleAuth`, for the same reason that we added `Oauth2`, and `Compute`, so that we can advertise the approach:

```js
const auth = new google.auth.GoogleAuth(opts);
const client = auth.getClient();
```

This allows me to advocate this methodology in the README as the _safe_ approach to using the client going forward, also the API surface ends up being more consistent:

```js
const oauth2Client = new google.auth.OAuth2(
  YOUR_CLIENT_ID,
  YOUR_CLIENT_SECRET,
  YOUR_REDIRECT_URL
);
```